lms/djangoapps/course_api/blocks/api.py

```diff
@@ -44,6 +45,8 @@ def get_blocks(
             which blocks to return their student_view_data.
         return_type (string): Possible values are 'dict' or 'list'. Indicates
             the format for returning the blocks.
+        block_types (list): Optional list of names of block types which to
+            return blocks.
```

Comment:
Can you update to something like this?
```
block_types (list): Optional list of block type names used to filter the final result of returned blocks.
```
@nasthagiri Is this a reasonable description?